pyproject.toml

```diff
@@ -10,9 +10,23 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "crick"
+description="High performance approximate and streaming algorithms"
+maintainers = [{name = "Jim Crist", email = "crist042@umn.edu"}]
+license = {text = "BSD-3-Clause"}
+readme = "README.rst"
 requires-python = ">=3.8"
 dynamic = ["version"]
 
+[project.urls]
+Source = "https://github.com/dask/crick"
```

Comment:
Is `Source` a valid option?
Maybe `Home` (just based from reading this https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#urls)

Reply:
Nvm, I see it's used in dask/distributed.